src/testRunner/_namespaces/compiler.ts

```diff
@@ -1,3 +0,0 @@
-/* Generated file to emulate the compiler namespace. */
-
-export * from "../../harness/_namespaces/compiler.js";

```

Comment:
Why do we need to remove this but keep `ts.server.ts` in the same folder ?

Reply:
It's referenced by other files, e.g. `src/testRunner/unittests/tsserver/exportMapCache.ts`